linphone/incoming-call.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# TODO see if we can drop the version
+ubuntu-app-launch linphone.cibersheep_linphone_0.4.2 linphone://incoming/$1
```

Comment:
ubuntu-app-launch `ubuntu-app-triplet linphone.cibersheep_linphone` linphone://incoming/$1

Reply:
Awesome, I didn't know that, thanks!